### DISCUSSION TITLE
Error saving configuration data

### DISCUSSION THREAD

**Discussion by jossmanbr2022** [pocketbase/pocketbase#670] [Q&A]
I try to change the name, url and logs max days retention of my application from the settings tab and it doesn't work, does anyone else have the same error?

* there is also an error when trying to rename a collection as it is not modified (later edited)

**Comment by ganigeorgiev** [comment]
What is the error? Could you check the response in the browser's "Inspect Tool > Network" tab?

You could also try starting the PocketBase executable with `--debug` and post here the error log messages (or look for them in the Admin UI > Logs).

**Reply by jossmanbr2022** [reply]
The fetch request is resolved when trying to make the change example I change the name of my application to AppDemo and it appears that it was modified correctly but the name Acme that comes by default continues to appear and so with all the configurations that I try to do.
The following message appears in the browser console: Autofocus processing was blocked because a document already has a focused element.
Even with the new version 7.7

**Reply by ganigeorgiev** [reply]
@jossmanbr2022 You could ignore the autofocus warning. I'm not able to reproduce the settings issue - I've tried it both locally and on the live demo.

Are you testing locally or on some public server?

If it is on a public server make sure that it has a persistent storage (Heroku dynos for example will not work).

**Reply by jossmanbr2022** [reply]
Thank you for your attention I will verify since I am using openport.io maybe that is why it does not work
